neurom/features/morphology.py

```diff
@@ -71,6 +71,11 @@
 feature = partial(feature, namespace=NameSpace.NEURON)
 
 
+def _return_1(*args, **kwargs):
```

Comment:
Not in favor of this. A lambda is much more readable for these trivial cases.

Reply:
Ah yeah, I needed it at some point because lambdas do not support `*args, **kwargs` but now it's no more needed. I will change.